core/tests/protractor/learnerDashboard.js

```diff
@@ -56,16 +60,9 @@ describe('Learner dashboard functionality', function() {
     browser.waitForAngular();
     element(by.css('.protractor-test-create-activity')).click();
     // Create new collection.
-    element(by.css('.protractor-test-create-collection')).click();
+    learnerDashboardPage.createExploration();
     browser.waitForAngular();
-    collectionEditor.addExistingExploration('14');
-    collectionEditor.saveDraft();
-    collectionEditor.closeSaveModal();
-    collectionEditor.publishCollection();
-    collectionEditor.setTitle('Test Collection');
-    collectionEditor.setObjective('This is a test collection.');
-    collectionEditor.setCategory('Algebra');
-    collectionEditor.saveChanges();
+    learnerDashboardPage.publishExploration();
```

Comment:
Ditto. This looks more like a job for the CollectionEditorPage.

Reply:
done, I have changed it